testnets/permtestnet/assetslist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "permtestnet",
+    "assets": [
+      {
+        "description": "Perm A Permissioned Blockchain",
+        "denom_units": [
+          {
+            "denom": "uperm",
+            "exponent": 0
+          },
+          {
+            "denom": "perm",
+            "exponent": 6
+          }
+        ],
+        "base": "uperm",
+        "name": "Perm",
+        "display": "perm",
+        "symbol": "PERM",
+        "logo_URIs": {
+          "png": "",
```

Comment:
actually, some images here would be ideal or else probably just removing the fields.